utils/run_test.sh

```diff
@@ -16,6 +16,8 @@
 #  Note: New tests should return non-zero if there are any
 #    failures.
 
+set -x
```

Comment:
Is it appropriate to enable _xtrace_ for this script on an ongoing basis or did you intend to remove this?

Reply:
I intend to remove this